README.md

```diff
@@ -164,11 +164,17 @@ class MyPage extends React.PureComponent {
 }
 ```
 
+### Path not found: 404
+
+It is optional to create a 404 page. If you want to create your own, add a `404.js` to `/pages`.
```

Comment:
Let's note the configuration option [`notFoundPath`](https://github.com/mapbox/batfish/blob/master/docs/configuration.md#notfoundpath).